src/pages/CategoryPage.js

```diff
@@ -60,6 +61,8 @@ class CategoryPage extends React.Component {
                 <Router.RouteHandler />
                 <mui.AppBar title={this.category.name} />
 
+                <CategoryHeader />
```

Comment:
In #67 I refactored HeaderBar to use for things like this. Because I think it's consistently the same styles.